platforms/core-configuration/configuration-cache/src/integTest/groovy/org/gradle/internal/cc/impl/fixtures/ExternalProcessFixture.groovy

```diff
@@ -97,7 +97,7 @@ class ExternalProcessFixture {
     private String getGroovyKotlinExecSpec() {
         return """
                 commandLine($commandLineAsVarargLiterals)
-                setStandardOutput(baos)
+                getStandardOutput().set(baos)
```

Comment:
💅 Can we use `=` here?

Reply:
Unfortunately not since this spec is used to generated `.java` files

Reply:
Actually no, this is GroovyKotlinExecSpec, let me try